package.json

```diff
@@ -202,7 +201,7 @@
     "body-parser": "^1.19.0",
     "capitano": "^1.9.2",
     "chalk": "^3.0.0",
-    "chokidar": "^2.1.8",
+    "chokidar": "^3.3.1",
```

Comment:
Set to `^3.3.1` to match livepush's `package.json`. What actually got installed and recorded in `npm-shrinkwrap` is `v3.4.2`.